packages/core/src/components/private/BaseSelect.tsx

```diff
@@ -36,7 +36,7 @@ class BaseSelect extends React.Component<Props & WithStylesProps> {
           {children}
         </FormInput>
 
-        <div
+        <i
```

Comment:
why `i`? instead of `span` if you want it to be inline?

Reply:
`i` = icons.

Reply:
aah and i guess it wouldn't matter if the svg is italic

(i = italic)

Reply:
Honestly, I mainly did it so testing was easier hah, but I have noticed icon libraries use this pattern.